pom.xml

```
@@ -52,6 +61,25 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>4.22.0</version>
```

You can use 
```suggestion
            <version>${liquibase.version}</version>
```
spring-boot-starter-parent already provides the version of liquibase